MAUI/Cartesian-Charts/Selection.md

```diff
@@ -84,44 +88,125 @@ Cartesian chart provides support to select a point programmatically on a chart u
+
+In [SfCartesianChart](), we can perform Multiple Selection for both DataPointSelection and SeriesSelection by selecting more than one segment in a series or more than one series in the chart by setting [SelectionBrush]() color and ChartSelectionType as Multiple, else by setting the [SelectedIndexes]() value from the ViewModel and set ChartSelectionType as Multiple to either [DataPointSelectionBehavior]() or [SeriesSelectionBehavior]() for load time selection changes.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+<chart:SfCartesianChart>
+. . .
+    <chart:SfCartesianChart.BindingContext>
+        <model:ViewModel/>
+    </chart:SfCartesianChart.BindingContext>
+    <chart:SfCartesianChart.SelectionBehavior>
+        <chart:SeriesSelectionBehavior SelectionBrush="Green" Type = Multiple SelectedIndexes = "{Binding Indexes}" />
+    </chart:SfCartesianChart.SelectionBehavior>
+. . .
+</chart:SfCartesianChart>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfCartesianChart chart = new SfCartesianChart();
+. . .
+ViewModel model = new ViewModel();
+chart.BindingContext = model;
+
+SeriesSelectionBehavior selection = new SeriesSelectionBehavior();
+selection.SelectionBrush = Brush.Green;
+selection.SelectedIndexes = model.Indexes;
+selection.Type = ChartSelectionType.Multiple;
+chart.SelectionBehavior = selection;
+. . .
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Multiple SeriesSelection support in MAUI SfCartesianChart](Selection_images/maui_multi_seriesselection.PNG)
+![Multiple DataPointSelection support in MAUI SfCartesianChart](Selection_images/maui_multi_dataPoint_selection.png)
+
+N> The Multiple Selection in [DataPointSelectionBehavior]() can be performed by doing the same using the ChartSeries SelectionBehavior property.
+
+## ClearSelection Method
+
+Both the SelectionBehavior has a public method called [ClearSelection ()](), which resets all current Selection Behavior property values to their default values and resets the color of the selected element to default.
+
+{% tabs %}
+
+{% highlight c# %}
+
+SfCartesianChart chart = new SfCartesianChart();
+
+DataPointSelectionBehavior selection = new DataPointSelectionBehavior();
+
+ColumnSeries series = new ColumnSeries();
+series.SelectionBehavior = selection;
 chart.Series.Add(series);
-this.Content = chart;
+
+selection.ClearSelection();
 
 {% endhighlight %}
 
 {% endtabs %}
 
-![Initial selection rendering support in MAUI Chart](Selection_images/maui_chart_selection_with_index.png)
+N> The [ClearSelection ()]() method can be performed in [SeriesSelectionBehavior]() by using the SfCartesianChart SelectionBehavior property. 
 
 ## Events
 
-The following selection events are available in the [SfCartesianChart](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.SfCartesianChart.html?tabs=tabid-1).
+The following public Chart Selection Events are available in [ChartSelectionBehavior]() and are common for both [DataPointSelectionBehavior]() and [SeriesSelectionBehavior]().
```

Comment:
The following public Chart Selection Events are available in [ChartSelectionBehavior]() and are common for both [DataPointSelectionBehavior]() and [SeriesSelectionBehavior]():

Reply:
I have changed the punctuations as mentioned

Reply:
I have also resolved this punctuation change.